packages/evolution-generator/src/scripts/generate_survey.py

```
@@ -23,10 +23,74 @@
+
+    normalized_scripts = set()
+    for script_name in only_scripts.split(","):
+        script_key = script_name.strip()
+        if not script_key:
+            continue
+        if script_key not in SUPPORTED_SCRIPT_ALIASES:
+            valid_values = ", ".join(sorted(SUPPORTED_SCRIPT_ALIASES.keys()))
+            raise ValueError(
+                f"Unknown script '{script_key}' in --only argument. "
+                f"Supported values are: {valid_values}"
+            )
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)